packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/crop/CropImageContract.kt

```diff
@@ -0,0 +1,98 @@
+package expo.modules.imagepicker.crop
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import androidx.core.net.toUri
+import androidx.core.os.bundleOf
+import com.canhub.cropper.CropImage
+import com.canhub.cropper.CropImageActivity
+import com.canhub.cropper.options
+import expo.modules.imagepicker.ImagePickerOptions
+import expo.modules.imagepicker.MediaType
+import expo.modules.imagepicker.PickingSource
+import expo.modules.imagepicker.createOutputFile
+import expo.modules.imagepicker.toBitmapCompressFormat
+import expo.modules.imagepicker.toFileExtension
+
+internal class CropImageContract(
+  private val sourceUri: Uri,
+  private val options: ImagePickerOptions,
+  private val pickingSource: PickingSource
+): MediaPickerContract() {
+//  TODO(@bbarthec): do we need it anymore?
+//  val exifDataHandler = ExifDataHandler(sourceUri)
+//
+//  class ExifDataHandler(private val uri: Uri) {
```

Comment:
I think so. IIRC this is needed to copy EXIF tags from original to cropped image, when user set `allowsEditing: true`